homeassistant/components/alexa/entities.py

```diff
@@ -389,19 +392,19 @@ class GenericCapabilities(AlexaEntity):
     The choice of last resort.
     """
 
-    def default_display_categories(self):
+    def default_display_categories(self) -> list[str]:
         """Return the display categories for this entity."""
         if self.entity.domain == automation.DOMAIN:
             return [DisplayCategory.ACTIVITY_TRIGGER]
 
         return [DisplayCategory.OTHER]
 
-    def interfaces(self):
+    def interfaces(self) -> list[AlexaCapability]:
         """Yield the supported interfaces."""
         return [
             AlexaPowerController(self.entity),
             AlexaEndpointHealth(self.hass, self.entity),
-            Alexa(self.hass),
+            Alexa(self.entity),
```

Comment:
Alexa is expecting a `State` in the constructor, not `HomeAssistant`.
I think that is a bug, and I have adjusted the code throughout the file.

Reply:
Thnx